Classes/Event/GlossaryKeyEvent.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+declare(strict_types=1);
+
+namespace WebVision\Deepltranslate\Core\Event;
+
+use WebVision\Deepltranslate\Core\Domain\Dto\CurrentPage;
+
+final class GlossaryKeyEvent
+{
+    private ?string $glossaryId = null;
+
+    public function __construct(
+        private readonly string $sourceLanguage,
```

Comment:
See above, use `public constructor property promotion`for all property initializations.

Additionally, let's add a

```php
public ?string $glossaryId = null;
```

which eliminates the need to define the property manually but still optional. Helps with simulating the event in custom event listener tests to mock/simulate different event states with a simple constructor call.

Question - do we really need `$glossaryId` nullable ? Or can we make it string only and use an empty string as default, for example:

```php
public string $glossaryId = '';
```

Getter / setter would be obsolete when using public CPP and having the glossary id also as public CPP (not nullable).